### PR TITLE
Rangefinder processing decoupled from reading

### PR DESCRIPTION
**Pull request by DzikuVx**
Addresses #1923

### CODE REVIEW

File: src/main/sensors/rangefinder.c [hunk 1]
```diff
@@ -188,7 +188,7 @@ timeDelta_t rangefinderUpdate(void)
 /**
  * Get the last distance measured by the sonar in centimeters. When the ground is too far away, RANGEFINDER_OUT_OF_RANGE is returned.
  */
-int32_t rangefinderRead(void)
+void rangefinderRead(void)
```
Comment (digitalentity) [inline src/main/sensors/rangefinder.c]: Let's call it `rangefinderUpdate()` and let's remove `rangefinderCalculateAltitude()`. 
Update function should do all the updating and calculation. Raw and compensated altitude should be accessible by reading rangefinder variables (similar to gyro and acc)

**Review by digitalentity — COMMENTED** [review]
Overall - I think this is a right way to go, just a few changed and we should be able to merge it

File: src/main/sensors/rangefinder.c (hunk 1 above)
Comment (DzikuVx) [reply src/main/sensors/rangefinder.c]: We already have `rangefinderUpdate` that does something completely different. So let's call it `rangefinderProcess` instead

**Comment by DzikuVx** [comment]
@digitalentity it should be better now

**Review by digitalentity — APPROVED** [review]

**Comment by DzikuVx** [comment]
Flight tested, data is filtered as required, merging

**Pull request merged**